File: Beecrowd/Beginner/1435 - 1985/1858 - Theon's Answer/Python/1858_TheonsAnswer.py
numberOfPeople = int(input())
theonAnswer = input().split()

theonAnswerPosition = 0

for i in range(numberOfPeople):
    if int(theonAnswer[i]) < int(theonAnswer[theonAnswerPosition]):
        theonAnswerPosition = i

print(theonAnswerPosition + 1)

File: Beecrowd/1 - Beginner/1435 - 1985/1858 - Theon's Answer/1858_TheonsAnswer.c
#include <stdio.h>

int main() {
    int numberOfPeople;
    scanf("%d", &numberOfPeople);

    int theonAnswer, theonAnswerPosition = 0;
    scanf("%d", &theonAnswer);
    
    int minimum = theonAnswer;

    for (int i = 1; i < numberOfPeople; i++) {
        scanf("%d", &theonAnswer);
        
        if (theonAnswer < minimum) {
            minimum = theonAnswer;
            theonAnswerPosition = i;
        }
    }

    printf("%d\n", theonAnswerPosition + 1);

    return 0;
}
